alignak/objects/resultmodulation.py

```diff
@@ -111,8 +111,6 @@ def pythonize(self):
 
         :return: None
         """
-        # First apply Item pythonize
-        super(Resultmodulation, self).pythonize()
```

Comment:
You're right, seems this function never called in this class ;)

I will remove it